app/src/main/java/com/zeapo/pwdstore/UserPreference.kt

```diff
@@ -301,6 +301,8 @@ class UserPreference : AppCompatActivity() {
 
             findPreference<CheckBoxPreference>(PreferenceKeys.ENABLE_DEBUG_LOGGING)?.isVisible = !BuildConfig.ENABLE_DEBUG_FEATURES
 
+            findPreference<EditTextPreference>(PreferenceKeys.BIOMETRIC_AUTH_TIMEOUT)
```

Comment:
what purpose does this serve?

Reply:
I think i was using it earlier. Will remove.